src/media/css/image-deferrer.styl

```diff
@@ -4,7 +4,7 @@
 img {
     transition: opacity .1s;
 }
-.screenshot .deferred {
+.thumbnail .deferred {
```

Comment:
hmm, what more specific class name could we use? ```.thumbnail``` could easily collide down the road